packages/components/src/DataList/components/DataListTags/DataListTags.css

```diff
@@ -1,27 +1,27 @@
+.tagWrapper {
+  position: relative;
+}
+
 .tags {
   display: flex;
```

Comment:
position moves up to the wrapper because it was used to set the position of the tag count, which now lives outside of the `tags` div